internal/worker/worker.go

```diff
@@ -260,6 +280,12 @@ func (w *BaseWorker) scatter(ctx context.Context, cmds []*cmd.DiceDBCmd) error {
 	case <-ctx.Done():
 		return ctx.Err()
 	default:
+
+		requestID, err := GenerateRandomUint32()
```

Comment:
@soumya-codes Using random requestID for each request towards shards. This approach looks okay?

Reply:
Can we use the request-id generation logic from the `id` package? 
I was thinking, the request-id can be a simple field in the worker that we can increment for every new request.
Randomness may not be very efficient and auditing may not be very easy.